tornado/httputil.py

```diff
@@ -243,6 +244,86 @@ def __deepcopy__(self, memo_dict):
         return self.copy()
 
 
+_cookie_value_decode = Cookie.SimpleCookie().value_decode
+_cookie_attr_reserved = Cookie.Morsel._reserved
+_cookie_attr_flags = set(["secure", "httponly"])
+
+# part of Cookie library module
+# https://docs.python.org/2.7/library/cookie.html
```

Comment:
And what's the difference between this version and the stdlib's? 

RFC 2068 was obsoleted by RFC 2616 (and subsequently by 723X), and RFC 2109 was obsoleted by 2965 (and subsequently by 6265). As far as I can tell 6265 is the current operative standard that we should be approaching.